pandas/core/window/rolling.py

```diff
@@ -1661,7 +1663,7 @@ def skew(self, **kwargs):
     )
 
     def sem(self, ddof: int = 1, *args, **kwargs):
-        return self.std(*args, **kwargs) / (self.count() - ddof).pow(0.5)
+        return self.std(*args, **kwargs) / self.count().pow(0.5)
```

Comment:
don't you need to pass ddof?

Reply:
I would say yes, ddof should be passed. But when comparing the rolling (and also expanding) version of the sem function with that of the 'regular' sem (i.e. nansem) function which is invoked (i.e. when called from a dataframe or series), you can see that the behavior is different:
![image](https://user-images.githubusercontent.com/11252091/106716438-bb18df00-65fe-11eb-84ca-8686ea78927a.png)

Digging in to it, you can see that in the [nansem version of the function](https://github.com/pandas-dev/pandas/blob/8e2e98baa62de4629ed3240abb66be9cc1c3ead5/pandas/core/nanops.py#L941) does not pass ddof, while [the rolling version](https://github.com/pandas-dev/pandas/blob/8e2e98baa62de4629ed3240abb66be9cc1c3ead5/pandas/core/window/rolling.py#L1664) does. In a perfect world I would say that passing ddof is correct, and that the nansem function has a problem. But, at the same time, I think the nansem version of the one which is used more regularly and is more heavily "tested", and thus the behavior of the rolling sem should at least match it.

Please let me know if you see it differently :)

Reply:
update: Even though this is an inconsistency between the two versions of the 'sem' function, makeing this ddof change in either the rolling version, or the nanops version leads to a rabbit-hole of related tests that need fixing, all of which are out-of-context of datetimelike-window centering 🤦. Because of that, this is probably something which should be tackled in a follow-up issue. Therefore I've reverted to the previous state, and added a work-around to the test. Hope this makes sense!